### PR TITLE
add deploy

### PR DESCRIPTION
**Pull request by FlipperLP**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: RUNNERNAME
```
Comment (Coder-Tavi) [inline .github/workflows/deploy.yml:9]: mainserver-pez*

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    env:
+      token: ${{ secrets.TOKEN }}
+      DBpassword: ${{ secrets.DBPASSWORD }}
```
Comment (Coder-Tavi) [inline .github/workflows/deploy.yml:12]: DB_PASSWORD

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      DBrootPassword: ${{ secrets.DBROOTPASSWORD }}
```
Comment (Coder-Tavi) [inline .github/workflows/deploy.yml:13]: Why is a root password needed?

**Review by Coder-Tavi — CHANGES_REQUESTED** [review]
99% sure we also need to pass in the DB_USERNAME and DB_NAME envs

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (FlipperLP) [reply .github/workflows/deploy.yml:9]: edit yourself, havent set up my dev env. on my new os

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (FlipperLP) [reply .github/workflows/deploy.yml:12]: change it in the runner settings. or edit yourself, havent set up my dev env. on my new os

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (FlipperLP) [reply .github/workflows/deploy.yml:13]: required on first startup

**Comment by FlipperLP** [comment]
> 99% sure we also need to pass in the DB_USERNAM

they are not confidential. and usally called the same as the app.

**Comment by FlipperLP** [comment]
Please not before merging,t hat the DB volume has been already created: this mean the same passwords as already set up with the previous setup needs to be used and will not be changed with a new one. This can cause the bot not to login with the correct credentials.

**Review by Coder-Tavi — APPROVED** [review]
Sigh, time to see if this works

**Pull request merged**